roles/redpanda_broker/tasks/install-redpanda.yml

```diff
@@ -1,17 +1,50 @@
 ---
-- name: Download the Redpanda repo script
-  ansible.builtin.get_url:
-    url: "{{ redpanda_repo_script }}"
-    dest: "/tmp/redpanda_repo_script.sh"
-    mode: '0755'
+- name: Download and import Redpanda GPG Key
+  shell: |
+    curl -1sLf "{{ rp_key_deb }}" | gpg --dearmor > {{ rp_key_path_deb }}
+  when: ansible_os_family == 'Debian'
```

Comment:
I don't love it, but if you walk the commit history you can see I tried other solutions without success. I'm going to come back to it again in phase 3.

Reply:
Sounds good. Fine with this as is for now then.

The alternative is doing the curl part with native ansible builtins, and then just doing the gpg dearmor as a shell operation. Then you could gate the dearmor based on file creation time or something.

We should probably figure out if the dearmor is even necessary. I'm seeing conflicting advice on this and I'm not embedded enough in the debian way to know what's true.

Reply:
I rejected ansible builtin curl because it required a tmp file to pass into dearmor and that felt ugly. I suppose I could always go that route. 

Beyond that from what I understand we can do either dearmor or rename to asc. Rename to asc caused key match to fail so I'm not sure whether that info is good or not, but I tested it pretty thoroughly and couldn't get it to work. Curl into dearmor isn't ideal because on second run it repeats.